frontend/views/utils/vError.js

```diff
@@ -0,0 +1,34 @@
+import Vue from 'vue'
+
+// Register a global custom directive called `v-error`
+// to automatically display vuelidate error messages
```

Comment:
It would be nice to leave down a dummy example directly here to explain the usage.

Reply:
Maybe later if we turn this into a library, but for now I think the examples in the other files are sufficient